VirtoCommerce.Platform.Web/appsettings.json

```diff
@@ -16,7 +16,8 @@
 
     "VirtoCommerce": {
         "ActivationUrl": "https://virtocommerce.com/admin/api/licenses/activate/",
-        "LicenseFilePath": "~/App_Data/VirtoCommerce.lic"
+        "LicenseFilePath": "~/App_Data/VirtoCommerce.lic",
```

Comment:
Why is this path different? Starts with tilda?

Reply:
Ends with comma

Reply:
I meant 
why this setting has a path with leading tilda
"LicenseFilePath": "~/App_Data/VirtoCommerce.lic"
but this setting has path without leading tilda
"LicensePublicKeyPath": "App_Data\\public.key"